feconf.py

```diff
@@ -177,8 +177,8 @@
 # The current version of subtopics dict in the topic schema.
 CURRENT_SUBTOPIC_SCHEMA_VERSION = 1
 
-# The current version of the question schema.
-CURRENT_QUESTION_SCHEMA_VERSION = 1
+# The current version of the question.
+CURRENT_QUESTION_VERSION = 1
```

Comment:
This is not required for a normal model version, since this is not a universal constant. Each model will have its own value of version based on edits to it.

Reply:
Done